.github/workflows/continous-delivery.yml

```diff
@@ -120,6 +120,8 @@ jobs:
         run: |
           gcloud compute instance-templates create-with-container zebrad-${{ needs.versioning.outputs.major_version || env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
           --boot-disk-type=pd-ssd \
+          --image-project=cos-cloud \
+          --image-family=cos-stable \
```

Comment:
Once upon a time I could have sworn this was the default choice, maybe that changed

Reply:
I think it's changed:

> By default, debian-11 is assumed for this flag.

https://cloud.google.com/sdk/gcloud/reference/compute/instances/create-with-container#--image-family